packages/titus-backend/lib/config/index.js

```diff
@@ -56,7 +57,24 @@ module.exports = {
     poolSize: 10,
     idleTimeoutMillis: 30000
   },
-  underPressure: {},
+  underPressure: {
+    exposeStatusRoute: {
+      url: config.HEALTHCHECK_URL,
+      routeResponseSchemaOpts: {
+        version: { type: 'string' },
+        serverTimestamp: { type: 'string' },
+        memoryUsage: {
+          type: 'object',
+          properties: {
+            eventLoopDelay: { type: 'string' },
+            rssBytes: { type: 'number' },
+            heapUsed: { type: 'number' }
+          }
+        },
+        db: { type: 'string' }
+      }
+    }
```

Comment:
please use fluent-json-schema here

Reply:
@mcollina to clarify

Are you asking to update the name of the package from `fluent-schema` to `fluent-json-schema`?
Or are you asking to create a schema of the configuration?

Reply:
I believe he's asking to use the fluent-json-schema library to define this json schema

Reply:
@p16 here's an example

```
routeResponseSchemaOps: S.object().prop('version', S.string()).prop('serverTimestamp', ...
```

Reply:
> Are you asking to update the name of the package from fluent-schema to fluent-json-schema?


Also

Reply:
@mcollina @simoneb name updated and schema created